storage/innobase/xtrabackup/doc/source/xtrabackup_bin/working_with_binary_logs.rst

```diff
@@ -3,36 +3,24 @@
 Working with Binary Logs
 ========================
 
-The ``xtrabackup`` binary integrates with information that |InnoDB|
-stores in its transaction log about the corresponding binary log
-position for committed transactions. This enables it to print out the
-binary log position to which a backup corresponds, so you can use it
-to set up new replication replicas or perform point-in-time recovery.
+The xtrabackup binary integrates with the log_status table. This integration enables xtrabackup to print out the backup's corresponding binary log position, so you can use it to set up new replication replicas or perform point-in-time recovery.
 
 Finding the Binary Log Position
 --------------------------------
 
-You can find the binary log position corresponding to a backup once
-the backup has been prepared. This can be done by either running the
-|xtrabackup| with the :option:`--prepare` or
-:option:`--apply-log-only` option. If your backup is from a server
-with binary logging enabled, |xtrabackup| will create a file named
-:file:`xtrabackup_binlog_info` in the target directory. This file
-contains the binary log file name and position of the exact point in
-the binary log to which the prepared backup corresponds.
+You can find the binary log position corresponding to a backup once the backup has been taken. If your backup is from a server with binary logging enabled, xtrabackup creates a file named xtrabackup_binlog_info in the target directory. This file contains the binary log file name and position of the exact point in the binary log to which the prepared backup corresponds.
```

Comment:
```suggestion
You can find the binary log position corresponding to a backup once the backup has been taken. If your backup is from a server with binary logging enabled, ``xtrabackup`` creates a file named :file:`xtrabackup_binlog_info` in the target directory. This file contains the binary log file name and position of the exact point in the binary log to which the prepared backup corresponds.
```

Reply:
We are moving away from :file: - otherwise fixed.